1001-2000/LIT1473GadlaL.xml

```diff
@@ -84,7 +87,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                </bibl>
             </listBibl>
          </div>
-         <div type="edition">
+         <div type="edition" xml:id="recension2">
+            <note>Recension 2 provided after <bibl>
+               <ptr target="bm:Bausi2003Libanos"/>
+            </bibl></note>
             <div type="textpart" subtype="incipit" xml:lang="gez">
```

Comment:
I wonder if this is the real incipit. I see that the quote is found on p. 8; but before there seems to be some text, I wonder if §1 and §2 (p. 1-2 of the text edition) could provide incipit.

Reply:
sure, if you feel like typing :) i will then wait to merge